app/posts/2020-01-01-divide-a-design-history-into-different-sections.md

```diff
@@ -31,15 +31,14 @@ You can create a section by grouping related posts. You can do this in 2 ways; k
 
 Next, create a page that lists these related posts. You can do that by creating an index page.
 
-1. Within the subfolder created in the previous step, create a markdown file, with the same name as the folder. For example, for the folder app/posts/support-interface/, add the file app/posts/support-interface/support-interface.md
+1. Within the `posts` folder, create a markdown file for each section. For example, for the folder `app/posts/support-interface/`, add the file `app/posts/support-interface.md`.
```

Comment:
Why the `posts` folder and not the subfolder?

For the TRA design history they were in the subfolder?
https://github.com/DFE-Digital/tra-digital-design-history/blob/main/app/posts/get-an-identity/get-an-identity.md

Or does this simplify the required markdown?

Reply:
So when I tried following the instructions, creating a file in the folder with the same name as the folder didn't work - the resulting URL used the name twice (`get-an-identity/get-an-identity`) and didn't seem to show up on the homepage. Either I got something wrong, or Eleventy (or the govuk plugin?) has changed since the instructions.

I think intuitively the index page to the top level `posts/` folder makes more sense to me, as it's easier to spot, and follows the URL path a bit more closely.  The other traditional approach would be a `index.md` file within the subfolder, but that doesn’t seem to be the conventional Eleventy setup?

Reply:
`index.md` files in folders containing directory data cause all sorts of issues, as you typically need to override the tags and other data for errors not to occur. This is a major pain with Eleventy to be honest (and this is why @fofr would be using `override:tags:` in his collection).

Reply:
Ah! So I just tried this locally, and even when adding all these extra bits to the front matter:

```yaml
override:tags: []
eleventyComputed:
  eleventyNavigation:
    key: "{{ title }}"
    excerpt: "{{ description }}"
    parent: home
permalink: "service1/{% if pagination.pageNumber > 0 %}page/{{ pagination.pageNumber + 1 }}{% endif %}/"
```

...something didn't work, I get this error when starting the app:

```bash
1. Having trouble writing template: "public/service1/index.html" (via EleventyTemplateError)
2. (./app/_layouts/collection.njk)
  Error: Node does not exist: Service 1 (via Template render error)
```

Probably I'm missing something else...

Do we think it’d be simpler to switch to suggesting adding the collection pages in the `posts/` folder, or should we continue to keep it in the subfolder, and document all the overrides?

Reply:
> If the collection page is in posts/, does it just work, without any overrides?

Yeah, I don't think you need any overrides then, as the json file only applies tags and layouts etc to all the files in the subfolder, so it's a bit simpler...

Reply:
> Do we think it’d be simpler to switch to suggesting adding the collection pages in the posts/ folder

Yes.